### ISSUE TITLE
difficulties to access the filter options with mouse Resulted

### ISSUE THREAD

**Issue by c17johha** [HGustavs/LenaSYS#6520]
<img width="154" alt="menu" src="https://user-images.githubusercontent.com/49199825/57070802-d1cb2d00-6cd8-11e9-9e86-227bf749b17a.png">

When hovering the mouse to access the filter options, one must be fast with mouse movement unless the options will dissapear. It seems to be caused by space between the hovering logo and the list

**Comment by a17maxsj** [comment]
This was referenced in #6516

**Comment by a17anter** [comment]
After I tried different options with CSS-styles I noticed that not just the "tratt" is affected but also the "arrows". I couldn't get the settings to work for both since there is an extra offset between them which I haven't found. See pictures below where they have the same CSS-styling.
![Tratt](https://user-images.githubusercontent.com/49204317/57296158-9364b200-70cc-11e9-9800-d45ac02defc5.PNG)
![Arrows](https://user-images.githubusercontent.com/49204317/57296159-9364b200-70cc-11e9-9633-3b98c2fb0cee.PNG)
